### ISSUE TITLE
Improve link styles (a11y)

### ISSUE THREAD

**Issue by aviannamiller** [omeka/theme-bigpicture#42]
Underlines (or some other type of non-color dependent distinguishing style) should be added for inline links.

Link to body text contrast should be at least 3:1, it's currently 1.3:1.

**Comment by aviannamiller** [comment]
Tags browse view:
Tags in tag cloud are measuring out at a 3.6:1 contrast. Since font sizes can vary here, we need to aim for at least 4.5:1.
